mesonbuild/environment.py

```diff
@@ -377,8 +377,8 @@ def detect_cpu_family(compilers: CompilersDict) -> str:
         trial = 'x86_64'
     elif trial in {'sun4u', 'sun4v'}:
         trial = 'sparc64'
-    elif trial in {'mipsel', 'mips64el'}:
-        trial = trial.rstrip('el')
+    elif trial.startswith('mips'):
+        trial = 'mips'
```

Comment:
This no longer returns 'mips64' in situations where it used to, so I'm not sure how you expect this to work.

Reply:
Oh I forgot that I just tested the 32 bit, let me send another one soon.